bop_toolkit_lib/visualization.py

```diff
@@ -175,6 +169,11 @@ def vis_object_poses(
     for pose in poses:
         # Rendering.
         if isinstance(K, CameraModel):
```

Comment:
I think it's the same issue here: the isinstance call will crash before the `if not htt_available` if htt_available is False (since CameraModel is not defined in this case).
Here's what I would do:

```
if htt_available and isinstance(K, CameraModel):
...
elif isinstance(K, np.ndarray) and K.shape==(3,3):
...
else:
    raise ValueError("Camera model 'K' type {} should be either CameraModel or np.ndarray".format(type(K)))
```